### PR TITLE
Add note and message doc to opsgenie_config

### PR DESCRIPTION
**Pull request by josedonizetti**
https://github.com/prometheus/alertmanager/blob/master/config/notifiers.go#L300
https://github.com/prometheus/alertmanager/blob/master/config/notifiers.go#L306

### CODE REVIEW

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -405,8 +405,12 @@ api_key: <secret>
 # The host to send OpsGenie API requests to.
 [ api_host: <string> | default = global.opsgenie_api_host ]
 
+# Alert text limited to 130 characters
```
Comment (brian-brazil) [inline content/docs/alerting/configuration.md]: There's no truncation done in the code if that's an actual API limit.

Comments should end in a full stop.

Comment (josedonizetti) [reply content/docs/alerting/configuration.md]: @brian-brazil Got this text from the api documentation itself, where they say the limit exists. Should I remove it?
https://docs.opsgenie.com/docs/alert-api-deprecated

Comment (brian-brazil) [reply content/docs/alerting/configuration.md]: It's more that we should implement it on the alertmanager side, as we do for some other limited fields.

Comment (josedonizetti) [reply content/docs/alerting/configuration.md]: @brian-brazil Gotcha! Gonna create an issue there and work on it.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**